src/main/java/seedu/duke/Duke.java

```diff
@@ -0,0 +1,115 @@
+package seedu.duke;
+
+import java.security.InvalidParameterException;
+import java.util.Scanner;
+
+/**
+ * Main part of logic of Duke.
+ */
+public class Duke {
+
+    /**
+     * Holds the storage of data
```

Comment:
Should there be a punctuation at the end of the description? I have noticed similar issue in other places too.

Reply:
I see, thanks